src/rules/variableNameRule.ts

```diff
@@ -15,11 +15,14 @@
  * limitations under the License.
  */
 
+// tslint:disable object-literal-sort-keys
+
 import * as ts from "typescript";
 
 import * as Lint from "../index";
 
-const BANNED_KEYWORDS = ["any", "Number", "number", "String", "string", "Boolean", "boolean", "Undefined", "undefined"];
+const BANNED_KEYWORDS = new Set(["any", "Number", "number", "String", "string", "Boolean", "boolean", "Undefined", "undefined"]);
+const bannedKeywordsStr = Array.from(BANNED_KEYWORDS).map((kw) => `\`${kw}\``).join(", ");
```

Comment:
why initialize a Set with an array and the convert it back to an array? Wouldn't it make more sense to just use the same array for both?
Also IIRC a set does not guarantee any order, which could separate `Number` and `number` in the output for example.